Jointify/View/ContentView.swift

```diff
@@ -10,65 +10,69 @@
 import SwiftUI
 import AVKit
 
-// MARK: ContentView
+// MARK: - ContentView
```

Comment:
`MARK: - KlassenName` für Abschnittsübersicht, die in Xcode rechts angezeigt wird und bei komplexeren Dateien dann hilft. Der Minusstrich zieht noch ne Linie davor und gehört als erstes vor die Klasse der Datei und am Ende vor die Previews